jsonrpc/src/methods/history.json

```
@@ -125,27 +125,27 @@
     }
   },
   {
-    "name": "portal_historyRecursiveFindContent",
-    "summary": "Look up a target content key in the network",
+    "name": "portal_historyGetContent",
+    "summary": "Get content from the local database if it exists, otherwise look up the target content key in the network. After fetching from the network the content is validated and stored in the local database before being returned.",
```

```suggestion
    "summary": "Get content from the local database if it exists, otherwise look up the target content key in the network. After fetching from the network the content is validated and stored in the local database if storage criteria is met before being returned.",
```

Maybe we should be a little more specific, as we won't store it if the storage criteria aren't met.

---

Yes good point, will update.